docs/reference/datatiers.asciidoc

```diff
@@ -174,25 +212,58 @@ tier.
 [[data-tier-allocation]]
 === Data tier index allocation
 
-When you create an index, by default {es} sets
-<<tier-preference-allocation-filter, `index.routing.allocation.include._tier_preference`>>
+The <<tier-preference-allocation-filter, `index.routing.allocation.include._tier_preference`>> setting determines the tier index shards should be allocated to.
+
+When you create an index, by default {es} sets the `_tier_preference`
 to `data_content` to automatically allocate the index shards to the content tier.
 
 When {es} creates an index as part of a <<data-streams, data stream>>,
-by default {es} sets
-<<tier-preference-allocation-filter, `index.routing.allocation.include._tier_preference`>>
+by default {es} sets the `_tier_preference`
 to `data_hot` to automatically allocate the index shards to the hot tier.
 
-You can explicitly set `index.routing.allocation.include._tier_preference`
-to opt out of the default tier-based allocation.
+At the time of index creation, you can override the default setting by explicitly setting 
+the preferred value in one of two ways:
+
+- By using an <<index-templates,index template>>. Refer to <<getting-started-index-lifecycle-management,Automate rollover with ILM>> for details.
+- From within the <<indices-create-index,create index>> request body. 
+
+You can override this 
+setting after index creation by <<indices-update-settings,updating the index setting>> to the preferred 
+value. 
+
+In this setting, you can provide multiple tiers in order of preference to prevent indices from remaining unallocated if no nodes are available in the preferred tier.
```

Comment:
@shainaraskas could we consider leaving the example ... ? This was intended to answer a common Support question that users believe ILM was manually overrode or applied incorrectly because it didn't _just_ say `_tier_preference: data_cold`. This attempted to highlight the system does and the user can set a descending preference list. 

```
<<cold-tier,cold tier>> would state `data_cold,data_warm,data_hot`
```

Reply:
is that a default value when the cold tier is created, or is it just an example of a value you could set? if the latter, try:

```suggestion
In this setting, you can provide multiple tiers in order of preference to prevent indices from remaining unallocated if no nodes are available in the preferred tier. For example, you might set your `<<cold-tier,cold tier>>`'s `_tier_preference` to `data_cold,data_warm,data_hot`.
```

if the former,

```suggestion
In this setting, you can provide multiple tiers in order of preference to prevent indices from remaining unallocated if no nodes are available in the preferred tier. For example, <<cold-tier,cold tier>>`'s `_tier_preference` is set to `data_cold,data_warm,data_hot` by default.
```

Reply:
We're ballpark, thanks for sticking through this nuance with me. When ILM automatically migrates your data to cold it sets `data_cold,data_warm,data_hot`. But that's not when the cold tier is created vs when ILM moves the index to the cold tier. If that makes sense? I would think your former with the appendage "like ILM Migrate does for the cold phase" or something 🤔.

Reply:
like this? 

```suggestion
This setting also accepts multiple tiers in order of preference. This prevents indices from remaining unallocated if no nodes are available in the preferred tier. For example, when {ilm} migrates an index to the cold phase, it sets the index `_tier_preference` to `data_cold,data_warm,data_hot`.
```

Reply:
That's excellent, cheers!